bukuserver/server.py

```diff
@@ -61,26 +61,31 @@ def get_tiny_url(rec_id):
 
 
 _BOOL_VALUES = {'true': True, '1': True, 'false': False, '0': False}
-def get_bool_from_env_var(key: str, default_value: bool) -> bool:
+def get_bool_from_env_var(key: str, default_value: bool = False) -> bool:
     """Get bool value from env var."""
     return _BOOL_VALUES.get(os.getenv(key, '').lower(), default_value)
 
 
-def init_locale(app):
+def init_locale(app, context_processor=lambda: {}):
     try:  # as per Flask-Admin-1.6.1
         try:
             from flask_babelex import Babel
             Babel(app).localeselector(lambda: app.config['BUKUSERVER_LOCALE'])
         except ImportError:
             from flask_babel import Babel
             Babel().init_app(app, locale_selector=lambda: app.config['BUKUSERVER_LOCALE'])
+        app.context_processor(lambda: {'lang': app.config['BUKUSERVER_LOCALE'] or 'en', **context_processor()})
     except Exception as e:
+        app.jinja_env.add_extension('jinja2.ext.i18n')
+        app.jinja_env.install_gettext_callables(gettext, ngettext, newstyle=True)
         app.logger.warning(f'failed to init locale ({e})')
+        app.context_processor(lambda: {'lang': '', **context_processor()})
 
 
 def create_app(db_file=None):
     """create app."""
     app = FlaskAPI(__name__)
+    os.environ.setdefault('FLASK_DEBUG', ('1' if get_bool_from_env_var('BUKUSERVER_DEBUG') else '0'))
```

Comment:
The normal way to enable debug mode doesn't seem to be accessible, so I'm using the variable fallback here.